packages/edit-site/src/components/actions/view-actions.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { external } from '@wordpress/icons';
+import { addQueryArgs } from '@wordpress/url';
+
+/**
+ * Internal dependencies
+ */
+import { ACTION_TYPES } from './constants';
+
+export const viewPostAction = {
+	id: 'view-post',
+	label: __( 'View' ),
+	type: ACTION_TYPES.primary,
+	icon: external,
+	isEligible( post ) {
+		return post.status !== 'trash';
+	},
+	perform( post ) {
+		document.location.href = post.link;
+	},
+};
+
+export const postRevisionsAction = {
+	id: 'view-post-revisions',
+	label: __( 'View revisions' ),
+	type: ACTION_TYPES.secondary,
```

Comment:
What are your thoughts on `isQuickAction: boolean` instead of `type`? Or other way to put it: do you anticipate having tertiary/quaternary/etc. actions?

It sounds like `type` as it is here, should be fine – I'm mostly curious.

Reply:
Boolean sounds good for now. If we need more types in the future we can always change again. I'll update tomorrow.